src/OmniSharp.DotNetTest/Models/DebugTestClassGetStartInfoRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using OmniSharp.Mef;
+using OmniSharp.Models;
+
+namespace OmniSharp.DotNetTest.Models
+{
+    [OmniSharpEndpoint(OmniSharpEndpoints.V2.DebugTestsInClassGetStartInfo, typeof(DebugTestClassGetStartInfoRequest), typeof(DebugTestGetStartInfoResponse))]
+    class DebugTestClassGetStartInfoRequest :  Request
+    {
+        public string[] MethodsInClass { get; set; }
```

Comment:
If this is just a list of methods, what happens if someone passes methods that aren't all in the same class?